.travis.yml

```diff
@@ -35,15 +50,16 @@ install:
 #   Dist::Zilla::Plugin::TestMLIncluder
 #   Dist::Zilla::Plugin::TestML1Includer
 #   Dist::Zilla::PluginBundle::Git
+- [ ! -e test ] || cpan-install --coverage
```

Comment:
This doesn't look like valid yaml. In fact
```
$ curl -s https://raw.githubusercontent.com/ingydotnet/zilla-dist-pm/c8a6097070d5136ea6c14551790dafc91b4aba28/.travis.yml | ysh
YAML::XS::Load Error: The problem:

    did not find expected comment or line break

was found at document: 1, line: 53, column: 18
while scanning a block scalar at line: 53, column: 17
```

Reply:
Thanks! Fixed. No idea why Travis worked with it.